CLI/src/main/java/main/Drifty_CLI.java

```diff
@@ -55,12 +58,49 @@ public static void main(String[] args) {
             link = null;
             String name = null;
             String location = null;
+            yamlFilePath = Paths.get(Program.get(Program.DRIFTY_PATH), YAML_FILENAME).toAbsolutePath().toString();
             for (int i = 0; i < args.length; i++) {
                 switch (args[i]) {
                     case HELP_FLAG, HELP_FLAG_SHORT -> {
                         help();
                         System.exit(0);
                     }
+                    case ADD_FLAG -> {
+                        if (i + 1 >= args.length) {
+                            messageBroker.msgBatchError("No URL provided.");
+                            System.exit(1);
+                        }
+                        for (int j = i + 1; j < args.length; j++) {
+                            addUrlToFile(args[j]);
+                        }
+                        System.exit(0);
+                    }
+                    case LIST_FLAG -> {
+                        listUrls();
+                        System.exit(0);
+                    }
+                    case REMOVE_FLAG -> {
+                        if (i + 1 >= args.length) {
+                            messageBroker.msgBatchError("No line number provided for removal.");
+                            System.exit(1);
+                        }
+                        for (int j = i + 1; j < args.length; j++) {
+                            if ("all".equalsIgnoreCase(args[j])) {
+                                removeAllUrls();
+                            } else {
+                                removeUrl(args[j]);
+                            }
+                        }
+                        System.exit(0);
+                    }
+                    case GET_FLAG -> {
+                        // Logic to download URLs goes here
+                        System.out.println("Downloading URLs...");
+                        batchDownloadingFile = yamlFilePath;
+                        batchDownloader();
+                        removeAllUrls();
+                        System.exit(0);
```

Comment:
@Femi-lawal If possible, please implement this change. I think it would really be helpful. You can discuss with me about the change before you implement it :smile:.

Reply:
@SaptarshiSarkar12 I've implemented it

Reply:
Great @Femi-lawal :+1:!